app/controllers/carts_controller.rb

```diff
@@ -3,13 +3,15 @@ def index
    @cart_items = session[:cart]
     @ordered_foods = {}
     @total = 0
-    @cart_items.each do |food_id, qty|
+    @prep_total = 0
+    @cart_items.each do |food_id, qty, prep_time|
```

Comment:
Inconsistent indentation detected.<br>Favor `unless` over `if` for negative conditions.